### PR TITLE
Initial commit of Sitefinity Conductors project code

### PR DESCRIPTION
**Pull request by Stefanie899**
This PR would commit the initial state of the AndcultureCode.CSharp.Sitefinity.Conductors project into Github.

### CODE REVIEW

File: src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+        public DynamicContentRepositoryConductor(
+            ILogger<IDynamicContentRepositoryConductor> logger
+        )
+        {
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Gets live content from the dynamic module manager by type, and optionally applies a filter.
```
Comment (KevinBusch) [inline src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs]: Minor - no need for the comma

File: src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            var providerName = DefaultProviderName;
+
+            DynamicModuleManager dynamicModuleManager = DynamicModuleManager.GetManager(providerName);
+
+            if (dynamicModuleManager == null)
+            {
+                r.AddErrorAndLog(
+                    _logger,
+                    ERROR_FAILED_TO_GET_DYNAMIC_MODULE_MANAGER,
+                    $"Attempt to get dynamic module manager with provider name {providerName} in method {nameof(GetContentByType)} returned null or empty."
```
Comment (cspath1) [inline src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs]: Question: Should this error message include `or empty` considering the expression above only checks for the object to be null?

**Review by cspath1 — APPROVED** [review]

File: src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        /// <param name="contentType">String representation of the content type.</param>
+        /// <param name="filter">Optional filter to apply to result set.</param>
+        /// <returns></returns>
+        public IResult<IQueryable<DynamicContent>> GetLiveContentByType(
+            string contentType,
+            Expression<Func<DynamicContent, bool>> filter = null
+        ) => GetContentByType(contentType, ContentLifecycleStatus.Live, filter);
+
+        /// <summary>
+        /// Gets content from the dynamic module manager by type, and optionally applies a filter.
```
Comment (KevinBusch) [inline src/AndcultureCode.CSharp.Sitefinity.Conductors/Domain/DynamicContentRepositoryConductor.cs]: Minor: again, no need for comma

**Review by KevinBusch — APPROVED** [review]
Looks good!

**Comment by brandongregoryscott** [comment]
@all-contributors add @Stefanie899 for code, doc

**Comment by brandongregoryscott** [comment]
@all-contributors add @KevinBusch for review

**Pull request merged**